EmailNotificationActivity/handler.ts

```diff
@@ -146,7 +148,8 @@ export const getEmailNotificationActivityHandler = (
 
   // trigger email delivery
   // see https://nodemailer.com/message/
-  const sendResult = await sendMail(lMailerTransporter, {
+
+  sendMail(lMailerTransporter, {
```

Comment:
I guess this returns a promise, shouldn't we await here?

Reply:
@gunzip no, `sendMail` changed. Now it returns a `TaskEither`

Reply:
then you need to call `run()` + `await` otherwise this would be a no-action (should be caught by some unit test)